scripts/docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+COMPOSE_VERSION=2.1.1
+
+command_available() {
+  type "$1" >/dev/null 2>&1
+}
+
+if ! command_available docker
+then
+    if ! command_available wget && command_available yum; then
+    sudo yum install wget
+    fi
+    wget -qO- https://get.docker.com/ | sh
+    sudo usermod -aG docker $USER
```

Comment:
This is generally considered a security error.

Reply:
adding the user to `docker` group?